assessment_framework_eng.md

```diff
@@ -17,14 +17,14 @@ The evaluation is composed of the following 6 sections:
 
 ---
 
-### Section 1 - Protecting personal or confidential data
+### Section 1 - Protecting personal or confidential data and comply with regulatory requirements
 
-**[Data privacy]**
+**[Data privacy and regulatory compliance]**
 
-The use of personal or confidential data carries the risk of exposure of such data, which can have very detrimental consequences for the producers, controllers or subjects of such data. Particularly in data science projects, they must therefore be protected and the risks of their leakage or exposure must be minimised.
+The use of personal or confidential data carries the risk of exposure of such data, which can have very detrimental consequences for the producers, controllers or subjects of such data. Particularly in data science projects, they must therefore be protected and the risks of their leakage or exposure must be minimised. Additionnally, AI models themselves can be attacked and must be protected. Finally, regulatory requirements specific to AI systems but be identified, known, and the data science activities of the organization must be compliant.
```

Comment:
what do you have in mind by "AI models themselves can be attacked"?

Reply:
That's an opening to evaluation elements 1.7 and 1.8 on model security (e.g. membership attacks etc.)

Reply:
Ok makes sense!